processor.go

```diff
@@ -65,6 +65,7 @@ func newProcessor(c config) *processor {
 		WriteTimeout:       c.Timeout,
 		MaxConnWaitTimeout: 1 * time.Second,
 		MaxConnsPerHost:    64,
+		DialDualStack:      c.SupportIPv6,
```

Comment:
```suggestion
		DialDualStack:      c.EnableIPv6,
```

Reply:
Changed name to `EnableIPv6` and `enable_ipv6`